src/main/java/org/dcsa/core/events/model/ActiveReeferSettings.java

```diff
@@ -0,0 +1,65 @@
+package org.dcsa.core.events.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.dcsa.core.events.model.enums.TemperatureUnit;
+import org.dcsa.core.events.model.transferobjects.ActiveReeferSettingsTO;
+import org.dcsa.core.model.AuditBase;
+import org.dcsa.core.model.GetId;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.relational.core.mapping.Column;
+
+import javax.validation.constraints.Size;
+import java.util.UUID;
+
+@NoArgsConstructor
+@Data
+public class ActiveReeferSettings extends AuditBase implements GetId<UUID> {
```

Comment:
```suggestion
public class ActiveReeferSettings extends AuditBase {
```

`GetId<UUID>` is generally redundant.  Is there a particular use for it here (otherwise I would prefer we removed it).

(There are tons of cases of `GetId<...>` sprinkled in the code base for historical reasons. Lets avoid spreading it further)

Reply:
fixed